File: CustomerManagement/src/main/java/com/gmlsoftware/model/client/ClientRequest.java
package com.gmlsoftware.model.client;

import java.util.Date;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class ClientRequest {
	private String firstName;
	private String middleName;
	private String lastName;
	private String secondLastName;
	private String emailAddress;
	private String phoneNumber;
	private Date startDate;
	private Date endDate;
}

File: CustomManagementFront/src/app/model/client/client-request.ts
export class ClientRequest {
    constructor(
        public clientID: number = 0,
        public firstName: string = '',
        public middleName: string | null = null,
        public lastName: string = '',
        public secondLastName: string | null = null,
        public emailAddress: string | null = null,
        public phoneNumber: string | null = null,
        public startDate: Date | null = null,
        public endDate: Date | null = null
    ) {}
}